packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -303,6 +323,7 @@ const generateComponentMeta = (): void => {
           type: propType.replace(/(?:BreakpointCustomizable|SelectedAriaAttributes)<(.+?)>/, '$1').trim(), // contains trailing space
           defaultValue: cleanedValue,
           ...(jsdoc?.match(/@deprecated/) && { isDeprecated: true }),
+          ...(jsdoc?.match(/__experimental__/i) && { isExperimental: true }),
```

Comment:
I think we should stay consistent with the annotation format

Reply:
Yes, good catch, this is just bold text in the end.
I think sticking to `@experimental` is the way to go since IDEs (at least WebStorm and VSCode) can handle it and have better formatted tooltips.
<img width="605" alt="Bildschirmfoto 2024-01-11 um 11 23 44" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/69471a71-9f87-469c-b0ab-37bd9201a4f1">
<img width="623" alt="Bildschirmfoto 2024-01-11 um 11 25 02" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/abd2009a-7e46-4663-8614-744c70e1c0a7">


vs 
<img width="569" alt="Bildschirmfoto 2024-01-11 um 11 24 05" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/a8d38708-b720-4d67-961c-20265d5565f0">
<img width="606" alt="Bildschirmfoto 2024-01-11 um 11 25 11" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/46094ea9-ec00-4231-9f3f-1eeb45adfc36">

Bold text even has negative impact on the docs generation.
<img width="1238" alt="Bildschirmfoto 2024-01-11 um 11 29 50" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/1e450f1c-2a77-4725-abd7-facf08854264">